app/controllers/admin/users_controller.rb

```diff
@@ -1,6 +1,12 @@
 class Admin::UsersController < ApplicationController
   before_action :admin?
 
+  def index
+    @users = Trainer.all + Trainee.all
```

Comment:
```suggestion
    users = Trainer.all + Trainee.all
```
không lạm dụng biến instance nhé em